compiler/aarch64/codegen/TreeEvaluatorTable.hpp

```diff
@@ -394,21 +394,21 @@
     TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::callIfEvaluator ,	// TR::callIf		// Call symref if first child evaluates to true.  Other childrem are arguments to the call.
-    TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::iRegLoadEvaluator ,	// TR::iRegLoad		// Load integer global register
-    TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::aRegLoadEvaluator ,	// TR::aRegLoad		// Load address global register
-    TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::lRegLoadEvaluator ,	// TR::lRegLoad		// Load long integer global register
+    TR::TreeEvaluator::iRegLoadEvaluator, // TR::iRegLoad		// Load integer global register
+    TR::TreeEvaluator::aRegLoadEvaluator, // TR::aRegLoad		// Load address global register
+    TR::TreeEvaluator::lRegLoadEvaluator, // TR::lRegLoad		// Load long integer global register
     TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::fRegLoadEvaluator ,	// TR::fRegLoad		// Load float global register
     TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::dRegLoadEvaluator ,	// TR::dRegLoad		// Load double global register
-    TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::sRegLoadEvaluator ,	// TR::sRegLoad		// Load short global register
-    TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::bRegLoadEvaluator ,	// TR::bRegLoad		// Load byte global register
-    TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::iRegStoreEvaluator ,	// TR::iRegStore		// Store integer global register
-    TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::aRegStoreEvaluator ,	// TR::aRegStore		// Store address global register
-    TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::lRegStoreEvaluator ,	// TR::lRegStore		// Store long integer global register
+    TR::TreeEvaluator::iRegLoadEvaluator, // TR::sRegLoad		// Load short global register
+    TR::TreeEvaluator::iRegLoadEvaluator, // TR::bRegLoad		// Load byte global register
+    TR::TreeEvaluator::iRegStoreEvaluator, // TR::iRegStore		// Store integer global register
+    TR::TreeEvaluator::lRegStoreEvaluator, // TR::aRegStore		// Store address global register
+    TR::TreeEvaluator::lRegStoreEvaluator, // TR::lRegStore		// Store long integer global register
```

Comment:
Why not save the extra level of indirection and have this table reference `commonRegStoreEvaluator` directly instead, and delete the i,a,l, etc. versions?